### DISCUSSION TITLE
Custom asset manager with custom modal doesn't work with bg image property (style manager)

### DISCUSSION THREAD

**Discussion by jay2success** [GrapesJS/grapesjs#4766] [Q&A]
Hello,

Thanks for this amazing library.
I am using vue with grapesjs, i am using custom modal for asset manager. 
It can set the image component but does not work with background image on style manager property.
I checked the code and i believed its because bg image property is not component and its not selected by editor.
How to solve this issue?
Thanks

```
editor.Commands.add("open-assets", {
        run(editor, sender, opts, e) {
          editorAssetModal.show()
        }
      });

assetSelect(asset, complete) {
      const selected = editor.getSelected()
      if (selected && selected.is('image')) {
        selected.addAttributes({ src: asset })
        complete && this.editorAssetModal.hide()
      } 
    },

```

**Comment by artf** [comment]
Follow [this guide](https://grapesjs.com/docs/modules/Assets.html#customization) to properly connect a custom asset manager.
